Microsoft.Azure.CosmosRepository/src/ChangeFeed/InMemory/ChangeFeedItemArgs.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.CosmosRepository.ChangeFeed.InMemory
+{
+    internal class ChangeFeedItemArgs<TItem> where TItem : IItem
+    {
+        public IEnumerable<TItem> ItemChanges { get; set; }
```

Comment:
Does this need to be public `set`?
```suggestion
        public IEnumerable<TItem> ItemChanges { get; private set; }
```